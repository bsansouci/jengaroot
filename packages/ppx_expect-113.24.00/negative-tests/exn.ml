open! Core.Std

let%expect_test _ =
  [%expect {| hi ho |}];
  failwith "hi ho";
  [%expect {| it's off to work we go |}]
;;
