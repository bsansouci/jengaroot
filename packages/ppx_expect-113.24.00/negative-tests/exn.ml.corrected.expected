open! Core.Std

let%expect_test _ =
  [%expect {||}];
  failwith "hi ho";
  [%collector_never_triggered]
;;
